SpatialGDK/Source/SpatialGDKServices/Private/SpatialPackageManager.cpp

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "Async/Async.h"
+#include "SpatialPackageManager.h"
+#include "SpatialGDKServicesConstants.h"
+
+DEFINE_LOG_CATEGORY(LogSpatialPackageManager);
+
+FSpatialPackageManager::FSpatialPackageManager() {}
```

Comment:
Remove the constructor if only using default

Reply:
it is used in two functions

Reply:
Yeah the default contractor will be used but the compiler will create a default contractor for you. Alternatively you could put FSpatialPackageManager() =default in the header which is kinda cute.